realm/realm-library/src/main/cpp/io_realm_internal_OsRealmConfig.cpp

```diff
@@ -373,11 +373,6 @@ JNIEXPORT jstring JNICALL Java_io_realm_internal_OsRealmConfig_nativeCreateAndSe
             }
         }
 
-        if (!config.encryption_key.empty()) {
```

Comment:
Why did this go now?

Reply:
Thomas cleaned it up and it is now being read only from the normal RealmConfiguration instead of needing to be configured twice.